src/server/models/listItem/providers/helpers.ts

```diff
@@ -177,9 +178,7 @@ export function getListItemContactInformation(listItem: ListItem): {
   contactPhoneNumber: string;
 } {
   const contactName = get(listItem?.jsonData, "contactName");
-  const contactEmailAddress =
```

Comment:
the issue here is they just need to be flipped.  Use the emailAddress if populated otherwise use publicEmailAddress.  Note, emailAddress is an optional field

Reply:
Ah that's interesting, I was under the assumption `emailAddress` was a required field and `publicEmailAddress` was the optional one?

<img width="827" alt="Screenshot 2022-08-04 at 17 40 15" src="https://user-images.githubusercontent.com/1377253/182904987-f68f260d-bbac-4497-a6f0-ceb8b23186f1.png">

Reply:
all good, I this should be fine as it is.